Makefile

```diff
@@ -60,7 +60,7 @@ release:
   	  -DBUILD_LAME="${BUILD_LAME}"             \
   	  -DCMAKE_SKIP_RPATH="${NO_RPATH}"     ..; \
       ${BUILD_SYSTEM} lrelease;                             \
-      ${BUILD_SYSTEM} -j ${CPUS};                           \
+      ${BUILD_SYSTEM} -l ${CPUS};                           \
```

Comment:
Yes, same on Linux and Travis unfortunately. It seems to prevent additional jobs being queued to take over when the current job blocks for I/O.